packages/core/src/components/carousel/carousel.tsx

```diff
@@ -113,6 +110,13 @@ export class AtomCarousel {
     Object.assign(this.swiperEl, params)
   }
   render() {
+    const injectStyles = [
+      '.swiper-button-disabled { opacity: 0 !important}',
+      '.swiper-pagination-custom { display: flex; justify-content: center; column-gap: var(--spacing-base); } ',
+      `.swiper-button-next { width: var(--spacing-${this.navigationButtonSize}); height: var(--spacing-${this.navigationButtonSize}); }`,
+      `.swiper-button-prev { width: var(--spacing-${this.navigationButtonSize}); height: var(--spacing-${this.navigationButtonSize}); }`,
+    ]
```

Comment:
(question) why this approach?

Reply:
The swiper lib doesn't have a prop to change the navigation buttons size. 
Before `injectStyles` was a private prop. I dont know why, but I can't use a prop inside another prop, so this way I can manage the swiper classes make the buttons size dynamic.